pythonx/ncm2_pyclang.py

```diff
@@ -41,7 +42,21 @@ def args_from_cmake(filepath, cwd, database_paths):
                     if normpath(cmd_for) == filepath:
                         logger.info("compile_commands: %s", cmd)
                         args = _extract_args_from_cmake(cmd)
-                        return args, cmd['directory']
+
+                        fixed_args = []
+                        add_next = False
+                        for arg in args:
+                            if add_next:
+                                add_next = False
+                                fixed_args.append('-I' + resolve(arg))
+                            elif arg == '-I':
+                                add_next = True
+                            elif arg.startswith('-I'):
+                                fixed_args.append('-I' + resolve(arg[2:]))
+                            else:
+                                fixed_args.append(arg)
+
+                        return fixed_args, cmd['directory']
```

Comment:
At current master, as in https://github.com/ncm2/ncm2-pyclang/blob/364f6f7acb8dd16a5ed51d9a1a3599cb4e024c3a/pythonx/ncm2_pyclang_proc.py#L292, the header completion uses `cmd['directory']` for resolving the path.  I don't understand why we need `cfg_path` instead of `cmd['directory']`

Reply:
> I understood cmd['directory'] to mean the directory of the file,

hmm...

I guessed `cmd['directory']` is the place for the build system running  gcc command. 
https://sarcasm.github.io/notes/dev/compilation-database.html

But I didn't find explicit definition of `cmd['directory']`